tests/juice_shope_negative_leave_feedback.spec.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const assert = require('assert');
+const FeedbackPage = require('../pages/JuiceShop/feedbackPage');
+const captcha = '121123asdasd';
+const comment = 'leaved comment'
+
+describe('Negative check captcha', () => {
+    before(async () => {
+        await FeedbackPage.navigateToFeedbackPage();
+        await FeedbackPage.dissmissWelcomeModal();
+    });
+
+    it('Should type wrong captcha', async () => {
+        const errorMessage = FeedbackPage.errorCaptchaMessage;
```

Comment:
You should use `await` here because `FeedbackPage.errorCaptchaMessage` returns promise that should be resolved